docs/docs/domain.mdx

```diff
@@ -222,35 +258,27 @@ Make this a nice table instead.
       max_value:  100.0
   ```
 
-
-
 * **Defaults**
 
   `max_value=1.0`, `min_value=0.0`
 
-
-
 * **Description**
 
-  All values below `min_value` will be treated as `min_value`, the same
-  happens for values above `max_value`. Hence, if `max_value` is set to
-  `1`, there is no difference between the slot values `2` and `3.5` in
-  terms of featurization (e.g. both values will influence the dialogue in
-  the same way and the model can not learn to differentiate between them).
-
+  If `influence_conversation` is set to `true`, the assistant's behavior will change
+  depending on the value of the slot. All values below `min_value` will be treated as
+  `min_value`, the same happens for values above `max_value`.
+  Hence, if `max_value` is set to `1`, there is no difference between the slot values
+  `2` and `3.5`.
```

Comment:
How would this slot type be used in a story? Based on the description it sounds like it only matters whether the value is above max or below min, but is that true?

Reply:
The story will be different depending on the concrete value of the float. So the slot value is capped to the supplied min / max values and then everything within this range is different for the bot. Would writing
```
depending on the value of the number
```
make it clearer?

Reply:
```suggestion
  If `influence_conversation` is set to `true`, the assistant's behavior will change
  depending on the value of the slot. If the value is between `min_value` and `max_value`, the specific value of the number is used. All values below `min_value` will be treated as
  `min_value`, and all values above `max_value` will be treated as `max_value`.
  Hence, if `max_value` is set to `1`, there is no difference between the slot values
  `2` and `3.5`.
```

better explicit than confused?